token-lending/program/tests/helpers/mod.rs

```diff
@@ -1287,7 +1299,7 @@ pub fn add_oracle(
             panic!("Unable to locate {}", filename);
         }));
 
-        let mut pyth_price = pyth::load_mut::<pyth::Price>(pyth_price_data.as_mut_slice()).unwrap();
+        let mut pyth_price = load_mut::<PriceAccount>(pyth_price_data.as_mut_slice()).unwrap();
```

Comment:
I don't have the full context but I think you can use `load_price_account` from [`pyth_sdk_solana::state`](https://github.com/pyth-network/pyth-sdk-rs/blob/main/pyth-sdk-solana/src/state.rs#L432)

Reply:
unfortunately i need a mutable reference (see the timestamp field get set below)

Reply:
🤔 maybe you can clone/copy it (PriceAccount has both traits)